concourse/upload.sh

```diff
@@ -16,6 +17,6 @@ echo "$LOCAL_MIRROR_PUBLIC" > "/root/.ssh/id_ed25519.pub"
 # change the permissions on the keys
 chmod 0600 /root/.ssh/*
 # Upload to local mirror
-rsync -e "ssh -o StrictHostKeyChecking=no -i /root/.ssh/id_ed25519" -rv rocknsm-iso/ $LOCAL_MIRROR_USER@$LOCAL_MIRROR_HOST:/var/www/mirror/public/isos/$LOCAL_MIRROR_REPO/
+rsync -e "ssh -o StrictHostKeyChecking=no -i /root/.ssh/id_ed25519" -rv rocknsm-iso/ $LOCAL_MIRROR_USER@$LOCAL_MIRROR_IP:/var/www/mirror/public/isos/$LOCAL_MIRROR_REPO/
 # Sync to public mirror
 ssh -i /root/.ssh/id_ed25519 -o StrictHostKeyChecking=no admin@${LOCAL_MIRROR_HOST} 'rsync -rlvtP -e "ssh -i ~/.ssh/mirror_sync" /var/www/mirror/public/ mirror_sync@mirror.rocknsm.io:/var/www/mirror/ --delete'
```

Comment:
Pretty sure you will need to also update line 22

Reply:
good catch